src/index.html

```diff
@@ -10,7 +10,55 @@
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      class="form"
+      data-qa="big"
+      action="#"
+      method="post"
+    >
+      <label class="search">
```

Comment:
`label` is redundant

Reply:
"label" was replaced with "div" in the code

Reply:
> `label` is redundant

"label" was replaced with "div" in the code